MultiServer.py

```diff
@@ -1530,14 +1530,13 @@ def get_hints(self, input_text: str, for_location: bool = False) -> bool:
         if hints:
             new_hints = set(hints) - self.ctx.hints[self.client.team, self.client.slot]
             old_hints = set(hints) - new_hints
-            if old_hints:
+            if old_hints and not new_hints:
                 self.ctx.notify_hints(self.client.team, list(old_hints))
-                if not new_hints:
-                    self.output("Hint was previously used, no points deducted.")
+                self.output("Hint was previously used, no points deducted.")
             if new_hints:
                 found_hints = [hint for hint in new_hints if hint.found]
                 not_found_hints = [hint for hint in new_hints if not hint.found]
-
+                old_hints = [hint for hint in old_hints]
```

Comment:
This could just be `list(old_hints)` at that point. Unless you feel strongly about it being the same format as the two previous lines

Reply:
There is also an argument for just wrapping a `list()` around `old_hints = set(hints) - new_hints` further up, because `old_hints` being a set is never taken advantage of, and then `old_hints` doesn't change types

Reply:
I only did it that way because the other two were doing it that way. I'd be fine with changing it when home (so I can make sure it doesn't break)